sql/engines/doris.py

```diff
@@ -0,0 +1,316 @@
+# -*- coding: UTF-8 -*-
+from sql.utils.sql_utils import get_syntax_type, remove_comments
+from .models import ResultSet, ReviewResult, ReviewSet
+from common.utils.timer import FuncTimer
+from common.config import SysConfig
+from . import EngineBase
+import traceback
+import pymysql
+import sqlparse
+import logging
+import re
+
+
+logger = logging.getLogger("default")
+
+class DorisEngine(EngineBase):
```

Comment:
```suggestion
class DorisEngine(MysqlEngine):
```

试试直接继承mysql，只用覆盖一些有差别的